frontends/election-manager/src/screens/tally_screen.tsx

```diff
@@ -163,6 +163,9 @@ export function TallyScreen(): JSX.Element {
               {hasAnyFiles ? (
                 <React.Fragment>
                   <tr>
+                    <TD as="th" narrow nowrap textAlign="right">
+                      #
+                    </TD>
```

Comment:
This column seems unnecessarily wide. I'm not sure if there's any way to shrink it to fit the content, but that might look better.

Reply:
probably because of misalignment of bottom row, can't believe i missed that.